packages/provider/src/utils.ts

```diff
@@ -15,9 +16,13 @@ export const messageToBytes = (message: BytesLike): Uint8Array => {
   return ethers.utils.toUtf8Bytes(message)
 }
 
-export const prefixEIP191Message = (message: BytesLike): Uint8Array => {
+export const prefixEIP191Message = (message: BytesLike, origin: string | undefined): Uint8Array => {
```

Comment:
Is there a reason you opted for `origin: string | undefined` instead of `origin?: string`? Yours will force an API break while the latter doesn't, but I'm not sure if you intended for that to force consumers to consider this.

Reply:
honestly, i would love to remove origin. let me see if i can match DCL messages in another way.